main.js

```diff
@@ -0,0 +1,111 @@
+// Ex 1: Thuế và lương ròng
+const getSalery = (salery) => {
+    if(salery >= 15000000){
+        console.log('Thuế thu nhập =',salery*30/100);
+        console.log('Lương ròng = ',salery*70/100);
+    }
+    else if (salery < 15000000 && salery > 7000000) {
```

Comment:
Nhìn kỹ cách viết if else
```suggestion
    } else if (salary > 7000000) {
```
ở trên đã kiểm tra salary >= 15m rồi, thì làm gì có chuyện nhảy được xuống case dưới này, mà phải kiểm tra salary < 15m, ở case này kiểu gì `salary` chả `bé hơn 15m` mà phải đi kiểm tra, tương tự cho các phần bên dưới